scripts/lib/upgrade-zulip-stage-3

```diff
@@ -179,6 +180,28 @@ subprocess.check_call(
     [os.path.join(deploy_path, "scripts", "setup", "generate_secrets.py"), "--production"]
 )
 
+# Adjust Puppet class names for the manifest renames in the 4.0 release
```

Comment:
This should get a `TODO/compatibility` comment for when we can remove it.

Reply:
Fixed the version in the comment, too!